UnitySDK/Assets/ML-Agents/Scripts/Grpc/RpcCommunicator.cs

```diff
@@ -424,6 +425,53 @@ private void HandleOnPlayModeChanged(PlayModeStateChange state)
             }
         }
 
+        private void CacheBrainParameters(string brainKey, BrainParameters brainParameters)
+        {
+            if (m_sentBrainKeys.Contains(brainKey))
+            {
+                return;
+            }
+
+            m_unsentBrainKeys[brainKey] = brainParameters;
+        }
+
+        private CommunicatorObjects.UnityRLInitializationOutputProto GetTempUnityRlInitializationOutput()
```

Comment:
nit: `CommunicatorObjects` is already imported.

Reply:
Fixed (and cleaned up a few other places where it was being used).